_notices/rsn0031.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 31 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "Support for CUDA 12 in v23.06"
+notice_author: RAPIDS Ops
+notice_status: In Progess
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v23.06+"
+notice_created: 2023-04-20
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2023-04-20
+---
+
+## Overview
```

Comment:
Please note: we are doing our best, but I am still unsure if we are ready to commit to shipping CUDA 12 in RAPIDS 23.06. I would like to avoid issuing an RSN that we must retract. I'd rather add the RSN once we know for sure that we can ship. Perhaps we can separate this RSN into another pull request?

Additionally, when we are ready to ship, we should clarify if CUDA 12 support will be rolled out via conda packages, pip packages, Docker images, or just some of those methods.

Reply:
We should hold on publishing this one then perhaps. Besides, adding support requires much less advance notice than removing support.